contracts/acl/ACL.sol

```diff
@@ -242,17 +243,7 @@ contract ACL is IACL, TimeHelpers, AragonApp, ACLHelpers {
     * @return boolean indicating whether the ACL allows the role or not
     */
     function hasPermission(address _who, address _where, bytes32 _what, bytes memory _how) public view returns (bool) {
-        // Force cast the bytes array into a uint256[], by overwriting its length
-        // Note that the uint256[] doesn't need to be initialized as we immediately overwrite it
-        // with _how and a new length, and _how becomes invalid from this point forward
-        uint256[] memory how;
-        uint256 intsLength = _how.length / 32;
-        assembly {
-            how := _how
-            mstore(how, intsLength)
-        }
-
-        return hasPermission(_who, _where, _what, how);
+        return hasPermission(_who, _where, _what, ConversionHelpers.castBytesToUintArray(_how));
```

Comment:
Is there any downside if we take advantage of libraries syntax sugar to do sth like `_how.toUintArray()`? (ofc, declaring the usage of `ConversionHelpers` for `bytes`)

Reply:
I actually want to avoid using the syntax sugar for these helpers, to make them more explicit so there's less chance for mistakes.

Particularly, the `cast` nomenclature may be helpful to communicate that the passed in variable shouldn't be used anymore.

Reply:
I've now renamed these cast functions to be `dangerouslyCastX()` in 410ad80, similar to how React has [`dangerouslySetInnerHTML()`](https://reactjs.org/docs/dom-elements.html#dangerouslysetinnerhtml), to make the intent more explicit.

Reply:
I see and I liked the naming idea :)